easypost/easypost_object.py

```diff
@@ -183,6 +183,9 @@ def __repr__(self):
     def __str__(self):
         return self.to_json(indent=2)
 
+    def __eq__(self, other):
```

Comment:
I'd recommend putting a check here, like 
```python
if not isinstance(other, EasyPostObject):
     return False
````
to short-circuit the function if we're not comparing a non-EasyPostObject to this clearly EasyPostObject.